hack/envtest/_matrix/v1.34.0-alpha.1.yaml

```diff
@@ -0,0 +1,2 @@
+go: 1.24
+etcd: v3.6.1
```

Comment:
Looks like the default is v3.6.0: https://github.com/kubernetes/kubernetes/blob/v1.34.0-alpha.1/cmd/kubeadm/app/constants/constants.go#L329

Reply:
Thanks
Got it
Fixed

Reply:
(No rush) But not pushed yet, right? :)

Reply:
Sorry. I miss it
Fixed @sbueringer